roscala/src/test/scala/coop/rchain/rosette/TransitionSpec.scala

```diff
@@ -306,4 +306,38 @@ class TransitionSpec extends FlatSpec with Matchers {
     val end = VirtualMachine.executeSeq(codevec, start)
     end.ctxt.ctxt.rslt should (be(Fixnum(3)) or be(Fixnum(7)))
   }
+
+  "Executing bytecode from expression \"(+ 1.2 2.3)\"" should "result in RoFloat(3.5)" in {
+
+    /**
+      * litvec:
+      &   0:   {RequestExpr}
```

Comment:
Is there a reason why you used `&` here?

Reply:
I think it is a misclick